app/lib/LevelSetupManager.coffee

```diff
@@ -67,10 +67,11 @@ module.exports = class LevelSetupManager extends CocoClass
     if @level.usesConfiguredMultiplayerHero()
      @onInventoryModalPlayClicked()
      return
-
-    if @level.isType('course', 'course-ladder', 'game-dev', 'web-dev') or window.serverConfig.picoCTF
-      @onInventoryModalPlayClicked()
-      return
+  
+    if @level.isType('course','course-ladder', 'game-dev', 'web-dev') or window.serverConfig.picoCTF
+      if @level.isType('course') and not me.showHeroAndInventoryModals()
+        @onInventoryModalPlayClicked()
+        return
 
     if @level.isSummative()
```

Comment:
Can this be true for any classroom level? If yes, we would need to check the flag here as well.